<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>429</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="companion9x.qrc">
    <normaloff>:/images/preferences.png</normaloff>:/images/preferences.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="4" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Splash library behaviour</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="QComboBox" name="backLightColor">
     <item>
      <property name="text">
       <string>Blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Orange</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yellow</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QComboBox" name="channelorderCB">
     <property name="whatsThis">
      <string>Channel order

This is used by the templated to determine which channel goes to what number output.</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string notr="true">R E T A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R E A T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R T E A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R T A E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R A E T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R A T E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E R T A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E R A T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E T R A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E T A R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E A R T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E A T R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T R E A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T R A E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T E R A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T E A R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T A R E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T A E R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A R E T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A R T E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A E R T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A E T R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A T R E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A T E R</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="locale_QB"/>
   </item>
   <item row="6" column="1" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="libraryPath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="libraryPathButton">
       <property name="text">
        <string>Open Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1" rowspan="2" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="downloadVerCB"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="fw_dnld">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="FwInfo">
       <property name="text">
        <string>FwInfo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QComboBox" name="stickmodeCB">
     <property name="whatsThis">
      <string>Mode selection:

Mode 1:
  Left stick:  Elevator, Rudder
  Right stick:  Throttle, Aileron

Mode 2:
  Left stick:  Throttle, Rudder
  Right stick:  Elevator, Aileron

Mode 3:
  Left stick:  Elevator, Aileron
  Right stick:  Throttle, Rudder

Mode 4:
  Left stick:  Throttle, Aileron
  Right stick:  Elevator, Rudder

</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Mode 1 (RUD ELE THR AIL)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode 2 (RUD THR ELE AIL)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode 3 (AIL ELE THR RUD)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode 4 (AIL THR ELE RUD)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="SplashFileName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="splashLibraryButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="companion9x.qrc">
         <normaloff>:/images/library.png</normaloff>:/images/library.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SplashSelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Open Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Personal splash library</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="historySize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>40</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Firmware</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>340</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Default Channel Order</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Default Stick Mode</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>BackLight Color</string>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="startupCheck_fw">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Show Splash Screen</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="showSplash">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Custom TX splash screen</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Recent History Size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="startupCheck_companion9x">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Check for updates on startup</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="4">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="imageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>64</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="InvertPixels">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Invert Pixels</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="clearImageButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="companion9x.qrc">
         <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Check selected Fw updates</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Language (requires restart)</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="splashincludeCB">
     <item>
      <property name="text">
       <string>Include companion splashes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Only user defined splashes</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
  <zorder>label_2</zorder>
  <zorder>label_7</zorder>
  <zorder>label</zorder>
  <zorder>startupCheck_fw</zorder>
  <zorder>label_4</zorder>
  <zorder>showSplash</zorder>
  <zorder>label_8</zorder>
  <zorder>label_6</zorder>
  <zorder>label_13</zorder>
  <zorder>channelorderCB</zorder>
  <zorder>label_14</zorder>
  <zorder>label_5</zorder>
  <zorder>locale_QB</zorder>
  <zorder>historySize</zorder>
  <zorder>stickmodeCB</zorder>
  <zorder>label_3</zorder>
  <zorder>backLightColor</zorder>
  <zorder>startupCheck_companion9x</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>splashincludeCB</zorder>
 </widget>
 <resources>
  <include location="companion9x.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
